app/views/shared/_navigation.html.erb

```diff
@@ -4,25 +4,25 @@
   <% end %>
   <% if user_signed_in? %>
     <%= link_to box_requests_path, class: 'nav-icon' do %>
-      <img src="https://voices-of-consent-static-images.s3.amazonaws.com/Status.svg" title="Status" />
+      <i class="fas fa-user-clock"></i>
```

Comment:
Great work. I'm glad there are labels now. One recommendation:

for each of the `<i class="fas…></i>`, consider adding `aria-hidden="true" role="presentation"'. This will suppress for assistive tech (AT). There are times font-awesome fails to load, and instead a square appears. This way it will not be read to screen readers and other AT.

Reply:
Okay, thanks for the tip.

Reply:
@renatamarques97 a designer had contributed the prior icons. maybe we use these font-awesome ones for now until we can get the graphics from her separate from the text?

Reply:
sure, I made the changes so, feel free to merge or ask for another thing :D @maebeale